### PR TITLE
[Fix] Fix CI bug due to mmcv index

### PR DESCRIPTION
**Pull request by kennymckormick**

### CODE REVIEW

**Comment by innerlee** [comment]
`-q` seems unrelated.

**Comment by kennymckormick** [comment]
> `-q` seems unrelated.

just to check log, seems if -q added, nothing will be printed

**Comment by innerlee** [comment]
Is this caused by mmdet? The CI was successful before

**Comment by kennymckormick** [comment]
> Is this caused by mmdet? The CI was successful before

Seems the mmcv with wrong version is installed in the cuda CI, see 2 screenshots below:
image
image

Try to find out why now.

**Comment by innerlee** [comment]
Try to modify the cmd format first

image

File: mmaction/models/backbones/resnet3d.py [hunk 1]
```diff
@@ -999,5 +1000,5 @@ def train(self, mode=True):
                     m.eval()
 
 
-if 'mmdet' in dir():
+if mmdet_imported:
     MMDET_SHARED_HEADS.register_module()(ResNet3dLayer)
```
Comment (hellock) [inline mmaction/models/backbones/resnet3d.py:1004]: This line is too weird. The following is more straight forward.

```python
MMDET_SHARED_HEADS.register_module(ResNet3dLayer)
```

Comment (kennymckormick) [reply mmaction/models/backbones/resnet3d.py:1004]: However, registry.register_module() return a function, 
```python
@MMDET_SHARED_HEADS.register_module()
class ResNet3dLayer():
    pass
```
is equivalent to: 
```python
MMDET_SHARED_HEADS.register_module()(ResNet3dLayer)
```

**Comment by innerlee** [comment]
CI has been fixed in master, so you can remove changes in build.yaml

**Comment by kennymckormick** [comment]
> CI has been fixed in master, so you can remove changes in build.yaml

Maybe you can still merge my modification. Since there is no built wheel for torch1.5.1 (which means you still can not pass the GPU CI), I modify build.yml to run CI under torch1.5.0 + torchvision 0.6.0.

**Comment by kennymckormick** [comment]
> Is this caused by mmdet? The CI was successful before

Think so. Bu

> CI has been fixed in master, so you can remove changes in build.yaml

Seems although https://download.openmmlab.com/mmcv/dist/cu101/torch1.5.1/index.html is an empty page, you can still install mmcv-full and pass the GPU CI. I'm not sure how does that happen, would you like to share the explanation with me?

**Comment by innerlee** [comment]
🤷 have no idea.
Ask @ycxioooong

**Comment by ycxioooong** [comment]
@kennymckormick There is no built wheel for torch1.5.1 from a long time ago. We may have a discussion on this issue offline.

**Comment by dreamerlin** [comment]
How is it going ? @kennymckormick

**Comment by kennymckormick** [comment]
> How is it going ? @kennymckormick

I think OK to merge.

**Review by innerlee — APPROVED** [review]

**Pull request merged**